### PR TITLE
[IMP] mail: introduce UseDragVisibleDropZone

### PR DESCRIPTION
**Pull request by alexkuhn**
Task-2831082

### CODE REVIEW

File: addons/mail/static/src/models/use_drag_visible_drop_zone.js [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+         */
+        dragCount: attr({
+            default: 0
+        }),
+        /**
+         * Determine whether the drop zone should be visible or not.
+         * Note that this is an observed value, and primitive types such as
+         * boolean cannot be observed, hence this is an object with boolean
+         * value accessible from `.value`
+         */
```
Comment (phenix-factory) [inline addons/mail/static/src/models/use_drag_visible_drop_zone.js:91]: Outdated comment :)
I think it safe to remove it now that it's a regular field.

**Review by phenix-factory — APPROVED** [review]
LGTM

File: addons/mail/static/src/models/use_drag_visible_drop_zone.js (hunk 1 above)
Comment (alexkuhn) [reply addons/mail/static/src/models/use_drag_visible_drop_zone.js:91]: I just kept the 1st sentence, which is still relevant.

**Comment by alexkuhn** [comment]
Moved to grouped PR: https://github.com/odoo/odoo/pull/89905

**Pull request closed without merging**